bin/node/runtime/src/lib.rs

```diff
@@ -917,6 +916,13 @@ impl_runtime_apis! {
 					steps,
 					repeat,
 				),
+				b"pallet-democracy" | b"democracy" => Democracy::run_benchmark(
```

Comment:
is `run_benchmark` coming from a trait and will it hence have the same signature for all modules? in that case it would be way easier to create a tuple of all modules that implement it and call `run_benchmark()` on the tuple. Or maybe even use `AllModules`.

Reply:
It doesn't have the same signature for all modules, because of instances. I'll see if it I can refactor it, in another PR.

Reply:
yeah might be a good improvement 👍  but not really impactful so maybe in the long future. having some duplicate code is fine for now.